fiori/app/admin/fiori-service.cds

```diff
@@ -85,7 +85,9 @@ annotate AdminService.Books.texts with @(
 
 // Add Value Help for Locales
 annotate AdminService.Books.texts {
-	locale @ValueList:{entity:'Languages',type:#fixed}
+	locale @ValueList:{entity:'Languages'};
+	locale @Common.ValueListWithFixedValues:true; //show as drop down, not a dialog
+	locale @Common.ValueListForValidation:''; // validate against this list
```

Comment:
This seems superfluous, at least doesn't make a difference for me.

Reply:
type:#fixed (line 88) was not doing what it was supposed to. Line 89, adds the functionality expected of #fixed.
You are right, with Line 89, Line 90 is superfluous. It is effective when 89 is not present.

Reply:
Thanks for the clarification